a8c-files.php

```diff
@@ -490,24 +504,55 @@ function delete_file( $file_name ) {
 	private function purge_file_cache( $url, $method ) {
 		global $file_cache_servers;
 
+		$parsed = parse_url( $url );
+		if ( empty( $parsed['host'] ) ) {
+			return $requests;
+		}
+
+		$uri = '/';
+		if ( isset( $parsed['path'] ) )
```

Comment:
Minor coding standards thing: we should always use braces, even for single line `if` clauses. Without them, we open up room for easy-to-make mistakes.

Reply:
Yeah, I just moved existing code and never changed it. I'll fix it.